### DISCUSSION TITLE
:thinking:  Feature Request: Minio support

### DISCUSSION THREAD

**Discussion by adriens** [duckdb/duckdb#6169] [Ideas]
As [`S3 Parquet Export`](https://duckdb.org/docs/guides/import/s3_export), would'n it be interesting to do the same into [`MinIO`](https://github.com/minio/minio) :grey_question:

**Comment by Mause** [comment]
We already support minio - in fact, it's what we use to run our s3 tests against

**Reply by adriens** [reply]
Maybe would it worth mentioning it explictely (at least for indexinx purpose on Google) :smile_cat:
